plugins/hikvision/src/main.ts

```diff
@@ -26,13 +26,20 @@ class HikVisionCamera extends RtspSmartCamera implements Camera {
                 events.on('close', () => ret.emit('error', new Error('close')));
                 events.on('error', e => ret.emit('error', e));
                 events.on('event', (event: HikVisionCameraEvent, cameraNumber: string) => {
-                    // if (this.getRtspChannel() && cameraNumber !== this.getCameraNumber()) {
-                    //     return;
-                    // }
                     if (event === HikVisionCameraEvent.MotionDetected
                         || event === HikVisionCameraEvent.LineDetection
                         || event === HikVisionCameraEvent.FieldDetection) {
-                        this.motionDetected = true;
+                        if (this.getRtspChannel() && cameraNumber !== this.getCameraNumber()) {
```

Comment:
is there a reason you uncommented this filter? there exists a filter in hikvision-api.ts, is that not working?